executor/linux/build.go

```diff
@@ -41,6 +41,12 @@ func (c *client) CreateBuild(ctx context.Context) error {
 		return fmt.Errorf("unable to upload build state: %v", c.err)
 	}
 
+	// setup the runtime build
+	c.err = c.Runtime.SetupBuild(ctx, c.pipeline)
+	if c.err != nil {
+		return fmt.Errorf("unable to create build %s: %w", c.pipeline.ID, c.err)
```

Comment:
Do you think this log message should reflect the function name?

```suggestion
		return fmt.Errorf("unable to setup build %s: %w", c.pipeline.ID, c.err)
```

Reply:
works for me. I was copying some of the other functions which are also inconsistent.